### PR TITLE
Bug/fix download url for slc file

### LINKED ISSUE #206: Fix download URL for SLC file

### PR DESCRIPTION
**Pull request by ckunki**
Closes #206

### CODE REVIEW

File: .github/workflows/build-and-publish.yml [hunk 1]
```diff
@@ -25,6 +25,12 @@ jobs:
       - name: Build Script Language Container (SLC)
         run: poetry run nox -s build_language_container
 
+      - name: PyPi Release
+        env:
+          POETRY_HTTP_BASIC_PYPI_USERNAME: "__token__"
+          POETRY_HTTP_BASIC_PYPI_PASSWORD: "${{ secrets.PYPI_TOKEN }}"
+        run: poetry publish
```
Comment (Nicoretti) [inline .github/workflows/build-and-publish.yml:32]: Is this part of the bugfix?

**Review by Nicoretti — DISMISSED** [review]

File: .github/workflows/build-and-publish.yml (hunk 1 above)
Comment (ckunki) [reply .github/workflows/build-and-publish.yml:32]: You are right, additionally (as part of the clean-up after the release) I moved the pypi release back to the old position in the workflow, which is before the GitHub release. 
I hope, this is OK for you?

Comment (tkilias) [reply .github/workflows/build-and-publish.yml:32]: But I am not ok with it, because a PyPi release doesn't make sense, if we couldn't upload the SLC

**Review by tkilias — CHANGES_REQUESTED** [review]
Revert the movement of the PyPi release

**Comment by ckunki** [comment]
> Revert the movement of the PyPi release

Reverted. Please see latest push.

**Review by tkilias — APPROVED** [review]

**Pull request merged**